rules/still-taking-manual-meeting-notes/rule.md

```diff
@@ -0,0 +1,68 @@
+---
+type: rule
+tips: ""
+title: Do you still take manual meeting notes?
+seoDescription: Stop wasting time and mental energy on manual meeting notes, use
+  AI-powered assistants like Read AI to automate your transcripts, summaries,
+  and follow-ups.
+uri: still-taking-manual-meeting-notes
+authors:
+  - title: Louis Roa
+    url: https://www.ssw.com.au/people/louis-roa/
+    img: https://www.ssw.com.au/people/static/Louis-Roa-Sketch-43386bcbae2db3e94bdb99b5bdae4f96.jpg
+created: 2025-05-09T14:25:00.000Z
+guid: 664492d1-464a-496d-a444-21da629e583b
+---
+![](bad_note.png "Do you still take manual meeting notes?")
```

Comment:
Hi @louisroa8189 

- [ ] 1. Don't start the rule with an image. Move it to the relevant section
- [ ] 2. Add the caption as per https://www.ssw.com.au/rules/rule/#6-images